script/test/test.sh

```diff
@@ -76,23 +76,19 @@ if [[ "$OSTYPE" != "msys" ]];
 then
     if [[ "$MODE" = "TEST" || "$MODE" = "ALL" ]];
     then
-        PYTHONPATH=./ pytest --cov-report term-missing:skip-covered  --cov-config=.coveragerc --cov-context=test --cov=eva/ -s -v --log-level=WARNING -m "not benchmark"
-    elif [[ "$MODE" = "RAY" ]];
```

Comment:
I think this causes Ray test to not run.

Reply:
Good catch!